linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -76,7 +76,17 @@ impl LineraNetConfig for LocalKubernetesNetConfig {
         );
         net.generate_initial_validator_config().await.unwrap();
         client.create_genesis_config().await.unwrap();
-        net.run().await.unwrap();
+        let result = {
+            match net.run().await {
+                Ok(value) => Ok(value),
+                Err(err) => {
+                    net.terminate().await.unwrap();
+                    Err(err)
```

Comment:
Why all the `unwrap()`s when the function returns `anyhow::Result<..>`? You should be able to use `?` or better `.context(...)?`

In this case, `return Err(error)` then you don't need to change the signature of `terminate`

Reply:
Also you don't need the `let result = { .. }; result.unwrap()`. `value` is `()` I guess?